test/xbgpu/test_engine.py

```diff
@@ -602,7 +613,8 @@ def heap_factory(batch_index: int) -> list[spead2.send.HeapReference]:
             # Or assert if incomplete_accs_total == incomplete_accums_counter * len(xbengine._pipelines)
             incomplete_accums_counter = 0
             base_batch_index = batch_start_index
-            for j in range(n_accumulations_completed[i]):
+            # for j in range(n_accumulations_completed[i]):
```

Comment:
Remove code rather than commenting it out.

Reply:
Argh, my apologies, thought I cleaned up properly.